definitions.md

```diff
@@ -161,20 +161,18 @@ A piece is a portion of a file that gets fitted into a sector.
 
 Security Parameter. Polling time is the time between two online PoReps in a PoSt proof.
 
+#### Power
+
+See `Power Fraction`.
+
 #### Power Fraction
 
-A miner's `Power Fraction` is the ratio of their committed storage over Filecoin's committed storage. It is used in leader election.
+A miner's `Power Fraction` or `Power` is the ratio of their committed storage as of their last PoST submission over Filecoin's total committed storage as of the current block. It is used in leader election.
```

Comment:
maybe "as of their last **expected** PoSt submission", as missing a PoSt changes the power too

Reply:
I think it might be confusing to distinguish between "expected PoSt" and "actual PoSt" we should just say PoSt submission when it happened and "failed to PoSt" or something when it didn't. 

So your change is great as is.